### ISSUE TITLE
FlaxDDIMScheduler is missing clip_sample implementation

### ISSUE THREAD

**Issue by SudeepDasari** [huggingface/diffusers#6901]
### Describe the bug

Setting `clip_sample=True` can sometimes help w/ numerical stability during DDIM inference. This is an important and useful feature that is actually promised in the FlaxDDIMScheduler [docstring](https://github.com/huggingface/diffusers/blob/97d004b9b4283d194a4d2c410be2d441527e0800/src/diffusers/schedulers/scheduling_ddim_flax.py#L87), and is implemented in the [pytorch version](https://github.com/huggingface/diffusers/blob/97d004b9b4283d194a4d2c410be2d441527e0800/src/diffusers/schedulers/scheduling_ddim.py#L131). However, its implementation is missing in the FlaxDDIMScheduler for some reason.

### Reproduction

This code should work but instead produces an error on `diffusers==0.26.2`:

```
>>> FlaxDDIMScheduler(clip_sample=True, ...)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/sudeep/anaconda3/envs/orca/lib/python3.10/site-packages/diffusers/configuration_utils.py", line 649, in inner_init
    init(self, *args, **init_kwargs)
TypeError: FlaxDDIMScheduler.__init__() got an unexpected keyword argument 'clip_sample'
```

### System Info

- `diffusers` version: 0.26.2
- Platform: Linux-5.4.0-166-generic-x86_64-with-glibc2.31
- Python version: 3.10
- Jax: 0.4.20 (w/ GPU)
- Huggingface_hub version: 0.20.3

**Comment by yiyixuxu** [comment]
hi:
I don't think it's a bug - but we could support it! 
do you want to open a PR to add this support?

**Comment by hi-sushanta** [comment]
can you please provide a full code?

**Comment by Bhavay-2001** [comment]
Hi @yiyixuxu, is this issue open for beginners?

**Comment by pcuenca** [comment]
Fixed by #7017
